rcp/org.testeditor.web.testexecutionlogview/login_test/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
For the Quickstart Guide we just needed a HTML Form to present the IDs just to recognize the Web-Elements and enter some input. This is the reason why we choosed this approach for the first step. 
For a further step we should communicate about what functionality logic we want for automation testing and should build a separate angular application.